### ISSUE TITLE
code

### ISSUE THREAD

**Issue by guoyanb** [CEN-KL/circ2DGNN#1]
The work is very interesting. When I conducted the experiments, I find that the loss function in github is not the same as the paper.  The code could not be download for Case Study. Could you please share them? Thanks
![Screenshot from 2024-12-15 15-19-19](https://github.com/user-attachments/assets/fb47515a-e5c9-450e-8230-71725ab88d10)
def compute_loss(pos_score, neg_score):
    # 间隔损失
    n_edges = pos_score.shape[0]
    return (1 - pos_score.unsqueeze(1) + neg_score.view(n_edges, -1)).clamp(min=0).mean()

**Comment by CEN-KL** [comment]
Thank you for your interest in our work and for pointing this out!
	1.The loss function provided in the code is consistent with the one described in the paper. Specifically, it follows the margin-based loss function commonly used for link prediction tasks. For more details, you can refer to the official Deep Graph Library (DGL) documentation on link prediction: https://docs.dgl.ai/en/2.2.x/guide/training-link.html
	2. The process of case study in our paper is to compute the dot product between the embeddings of specific circRNA and disease nodes and then rank the results accordingly.  With the GNN model trained by following the README file, you can  obtain the embeddings of circRNAs and diseases.

**Comment by guoyanb** [comment]
Are the results of the paper are generated by ’eval_test_set(G, G_test, neg_G_test)‘ ? and  what is the role of ’eval_test_set(G, G_test, neg_G_test)‘  ? Thanks

**Comment by CEN-KL** [comment]
> Are the results of the paper are generated by ’eval_test_set(G, G_test, neg_G_test)‘ ?  Yes
and what is the role of ’eval_test_set(G, G_test, neg_G_test)‘ ?  validate the model’s performance on an untrained test set :)
